user-guide/network/nictag.rst

```diff
@@ -11,6 +11,10 @@ For other use cases, it is possible to add following NIC tags:
 - internal
 - storage
 
+Administrator can also add new NIC tags (e.g. overlay, etherstub), these new NIC tags are then collected from the node and will be presented when creating network.
```

Comment:
- How about this: "Administrator can also add new NIC tags with custom name or type (normal, overlay or etherstub). These ... "
- And maybe some note about refreshing the node after adding a NIC tag